components/create-post-page/UploadTextbookCover.tsx

```diff
@@ -1,27 +1,27 @@
 import ImageUploadModal from "@components/ImageUpload/ImageUploadModal";
-import { ChangeEvent } from "react";
 
 interface Props {
+  key: string;
```

Comment:
I feel like you shouldn't have to do this, I think you just need to change the key value for the `UploadTextbookCover` component instead of passing this through as a prop.

Reply:
I realized a generated key is not needed at all for this, so I just gave it a general name. See here: https://github.com/calebmech/Book-Bazar/blob/85-edit-post-modal/components/create-post-page/UploadTextbookCover.tsx#L19